includes/class-field-manager.php

```diff
@@ -1,4 +1,4 @@
-<?php
+<?php declare(strict_types=1); 
```

Comment:
to follow the coding convention of this project, we will not use `declare(strict_types=1);`